feconf.py

```diff
@@ -28,6 +28,11 @@
 # code in core/platform.
 PLATFORM = 'gae'
 
+# This variable is for serving minified resources
+# when set to false. It allows minified resources to be used when
+# running the server in the dev environment.
+FORCE_DEV_MODE = True
```

Comment:
Hi @MAKOSCAFEE -- sorry, I think I might have gotten this the wrong way round in my other comment. DEV_MODE is "naturally" True, so what we actually want here is FORCE_PROD_MODE, I think. Sorry about that.

Reply:
Done